Sources/Sentry/SentrySystemWrapper.mm

```diff
@@ -99,7 +100,8 @@ - (NSNumber *)cpuEnergyUsageWithError:(NSError **)error
             ;
         }
     }
-    return @(powerInfo.cpu_energy.total_system + powerInfo.cpu_energy.total_user);
+    return @(powerInfo.task_energy);
```

Comment:
`m`: Can we somehow add a test for this? It's always a bit of a smell when we change something, and we don't have to adapt the tests. As far as I know there aren't any tests for this, but it would be great to have some.